test_scripts/RC/OnRemoteControlSettings/003_SUCCESS_in_case_AUTO_ALLOW_seq_3.lua

```diff
@@ -0,0 +1,56 @@
+---------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SDL_RC] Resource allocation based on access mode
+--
+-- Description: TRS: OnRemoteControlSettings, #1
+-- In case:
+-- 1) SDL received OnRemoteControlSettings notification from HMI with allowed:true
+-- 2) and "accessMode" = "AUTO_ALLOW" or without "accessMode" parameter at all
+-- 3) and RC_module on HMI is alreay in control by RC-application
+-- SDL must:
+-- 1) provide access to RC_module for the second RC_application in HMILevel FULL after it sends control RPC
+-- (either SetInteriorVehicleData or ButtonPress) for the same RC_module without asking a driver
+-- 2) process the request from the second RC_application
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonRC = require('test_scripts/RC/commonRC')
+
+--[[ Local Variables ]]
+local modules = { "CLIMATE", "RADIO" }
+local access_modes = { nil, "AUTO_ALLOW" }
+
+--[[ Local Functions ]]
+local function ptu_update_func(tbl)
+  tbl.policy_table.app_policies[config.application2.registerAppInterfaceParams.appID] = commonRC.getRCAppConfig()
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonRC.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonRC.start)
+runner.Step("RAI1, PTU", commonRC.rai_ptu, { ptu_update_func })
+runner.Step("RAI2", commonRC.rai_n, { 2 })
+
+-- App's HMI levels: 1 - BACKGROUND, 2 - FULL
```

Comment:
@dboltovskyi HMI level of both apps is NONE here. Please correct comment in code or delete it.